v2h_templates.py

```diff
@@ -43,6 +49,57 @@
 tr:hover td:nth-child(n+2) {
     filter: hue-rotate(-50deg);
 }
+td {
+  border-radius: 2px;
+}
+td div+div {
+    margin-top: 0.8em;
+}
+.tooltip {
+    font-weight: bold;
+    position: relative;
+    text-align: left;
+}
+.tooltip div {
+    display: inline-block;
+    width: 19px;
+}
+.tooltip:before {
+    content: attr(data-text);
+    position: absolute;
+    top: 100%;
+    transform: translateX(-50%);
```

Comment:
This property moves tooltip out of main container, when window width is less then 890px

```suggestion
    + left: 0;
```